domains/certificates/Query.API/API.UnitTests/MeasurementsSyncer/SlidingWindowServiceTest.cs

```diff
@@ -331,13 +332,13 @@ public void GivenMissingMeasurementInQuarterResolution_WhenUpdatingSlidingWindow
         Assert.Equal(newSynchronizationPoint, window.MissingMeasurements.Intervals[0].To);
     }
 
-    private Measurement CreateMeasurement(string gsrn, long from, long to, long quantity, bool quantityMissing)
+    private Measurement CreateMeasurement(Gsrn gsrn, long from, long to, long quantity, bool quantityMissing)
     {
         return new Measurement
         {
             DateFrom = from,
             DateTo = to,
-            Gsrn = gsrn,
+            Gsrn = gsrn.Value,
```

Comment:
Gsrn = gsrn 

change Measurement so it has an Gsrn type and not string

Reply:
Measurement is from gRPC protobuf file

Reply:
Good point :)